app/controllers/api/v1/routes_controller.rb

```diff
@@ -16,6 +16,11 @@ class Api::V1::RoutesController < Api::V1::BaseApiController
     properties[:route_stop_patterns_by_onestop_id] = entity.route_stop_patterns.map(&:onestop_id)
   }
 
+  RESPONSE_FORMATS_BY_ACTION = {
+    :index => [:json, :geojson],
+    :show => [:json, :geojson],
+  }
+
```

Comment:
Oops, I should remove this. A test to list out formats for specific endpoints and actions--but I wasn't able to access it from the code in `api_v1_controller.rb`.